backend/apps/kloudust/lib/cmd/scripts/exportVM.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Params:
+# {1} VM Name
+# {2} SFTP Username
+# {3} SFTP Host
+# {4} SFTP Password
+# {5} Remote destination directory
+# {6} SFTP Port 
+
+VM_NAME="{1}"
+SFTP_USER="{2}"
+SFTP_HOST="{3}"
+SFTP_PASS="{4}"
+DEST_DIR="{5}"
+SFTP_PORT="{6}"
+
+function exitFailed() {
+    echo Error: $1
+    echo Failed
+    exit 1
+}
+
+if [ -z "$VM_NAME" ] || [ -z "$SFTP_USER" ] || [ -z "$SFTP_PASS" ] || [ -z "$SFTP_HOST" ] || [ -z "$DEST_DIR" ] || [ -z "$SFTP_PORT"]; then
+    echo "Usage: $0 <vm-name> <sftp-user> <sftp-password> <sftp-host> <remote-dir> <sftp-port>"
+    exit 1
+fi
+
+for cmd in virsh sshpass ssh scp; do
+    if ! command -v "$cmd" >/dev/null 2>&1; then
+        exitFailed "❌ Error: '$cmd' is not installed. Please install it first."
```

Comment:
Should this ever happen? Isn't this implying host doesn't have software we need host to have? But host is initialized by the cloud. No one can install anything after that to the host. So (1) why this error check? did you encounter it? (2) if you encountered it did you modify the host init script to ensure these are ALWAYS there? if not then this is a CRITICAL ERROR AND MUST BE FIXED BEFORE THIS PULL IS ACCEPTED (3) If you did modify the host init script to ensure this (and I am assuming you didn't) then why this check in the first place?